src/WP_CLI/Entity/RecursiveDataStructureTraverser.php

```diff
@@ -171,6 +171,7 @@ protected function create_key() {
 	 * @return bool
 	 */
 	public function exists( $key ) {
-		return ( is_array( $this->data ) && array_key_exists( $key, $this->data ) ) || ( is_object( $this->data ) && property_exists( $this->data, $key ) );
+		return ( is_array( $this->data ) && array_key_exists( $key, $this->data ) ) ||
```

Comment:
In terms of readability, pushing the operator to the next line is preferable, i.e.:
```
if ( $condition1
     || $condition2
     || $condition3 ) {
}
```